### PR TITLE
Cli import multiple

### PR DESCRIPTION
**Pull request by richardgill**
Support importing multiple.

Tested with `--types` flag and inferring a multiple.

https://github.com/xataio/file-import-samples updated to include multiples

### CODE REVIEW

File: packages/importer/src/columns.ts [hunk 1]
```diff
@@ -84,14 +114,17 @@ export const guessColumnTypes = <T>(
   if (columnValues.every(isEmail)) {
     return 'email';
   }
+  if (columnValues.some(isMultiple)) {
```
Comment (richardgill) [inline packages/importer/src/columns.ts:117]: A good option might be to just never guess `multiple`

Comment (SferaDev) [reply packages/importer/src/columns.ts:117]: I'm okay not guessing it, if uploading with an existing schema works or passing the type explicitly works.

**Comment by SferaDev** [comment]
@kostasb Can you give it a try?

**Review by SferaDev — APPROVED** [review]

**Comment by kostasb** [comment]
Works well when explicitly specifying the type as multiple!

Not sure if we were planning to infer multiples or just create string columns if not specifying? It doesn't seem to infer multiple.

```
id,mymulti,mystring
1,"one,two",three

xataCli import csv my.csv --table=mytable1 --branch=main --columns=id,mymulti,mystring --types=string,multiple,string
// works!

xataCli import csv my.csv --table=mytable2 --branch=main
// mymulti is created as a string column
```

**Comment by SferaDev** [comment]
> It doesn't seem to infer multiple.

We discussed this with Richard and no infer for now as it will detect some false positives

**Pull request merged**